tests/unit/services/export/test_geojson.py

```diff
@@ -64,3 +64,39 @@ def test_kili_export_labels_geojson(mocker: pytest_mock.MockerFixture):
 
     assert output["type"] == "FeatureCollection"
     assert len(output["features"]) == 5  # 5 annotations in geotiff_image_project_assets.json
+
+
+def test_process_asset_with_external_id_containing_slash():
+    with TemporaryDirectory() as f:
```

Comment:
you can get a temp dir using pytest fixture: https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture